cd_database/migrations/1714726277027648_events.up.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS events
     timestamp TIMESTAMP,
```

Comment:
> This isn't done through a migration because sqlite does not support altering table schemas
That's fine for now, but next time we might have already rolled this version out somewhere. So we would need a different approach next time.